etc/docker/kubernetes/README.rst

```diff
@@ -0,0 +1,63 @@
+========================================
+Setting up Rucio in a Kubernetes cluster
+========================================
+
+
+Ingress configuration
+---------------------
+Depending on the platform hosting the Kubernetes cluster a variety of ingress controllers are available. The cluster currently used only supports Traefik so this readme only describes this but any other ingress controllers should work as well.
+
+First, to be able to serve https request a valid certificate/key is needed. Those have to be added to the cluster as a secret::
+
+    $ kubectl create secret tls tls-secret --key tls.key --cert tls.crt --namespace=kube-system
+
+Next, the provided traefic.toml configuration file has to be added to the cluster::
+
+    $ kubectl create configmap logging-configmap --from-file=ingress/traefik.toml --namespace=kube-system
+
+Then the daemonset configuration can be applied to the cluster::
+
+    $ kubectl apply -f ingress/traefik-ds.yaml
+
+This will create the traefik daemonset and mount the certificate and configuration file. The daemonset works with a node selector, i.e., the ingress controller will only be run on nodes that are labeled. You can label a node with::
+
+    $ kubectl label node <node-name> role=ingress
+
```

Comment:
it's not clear for me. Would you have one example or latter in the documentation ?

Reply:
what's not clear about it? there is an example.

Reply:
sorry -- I'm learning kubernetes at the same time.   `kubectl get nodes` gives the list of nodes on the cluster.